modules/manage/pages/kubernetes/troubleshooting/troubleshoot.adoc

```diff
@@ -20,10 +20,86 @@ To check the values that were set in the Redpanda Helm chart:
 
 [,bash]
 ----
-helm get values redpanda -n redpanda --all
+helm get values redpanda --namespace <namespace> --all
 ----
 
-== HelmRelease is not ready
+== View Redpanda logs
+
+Logs are sent directory to STDOUT for each Pod. Use the command below to view the logs for a Pod:
+
+```bash
+kubectl logs <pod-name> --namespace <namespace>
```

Comment:
I would advise to use 

```
kubectl logs --namespace <namespace> -l app.kubernetes.io/component=redpanda-statefulset,app.kubernetes.io/instance=redpanda,app.kubernetes.io/name=redpanda
```

Maybe the `app.kubernetes.io/component=redpanda-statefulset` is the best label to pin it.

Reply:
Is that label always the same no matter what the `overrideFullName` value is?

Reply:
https://github.com/redpanda-data/helm-charts/blob/9706db148194cbfdbc12aec82e086190d2ec64de/charts/redpanda/templates/_statefulset.tpl#L30

The `{{ (include "redpanda.name" .) | trunc 51 }}` dictates the name and yes `overrideFullName` can change this value.

https://github.com/redpanda-data/helm-charts/blob/9706db148194cbfdbc12aec82e086190d2ec64de/charts/redpanda/templates/_helpers.tpl#L21